kafka-daq-report/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[workspaces]
+resolver = "2"
```

Comment:
I added this line because I kept getting warnings in the terminal about there being a mismatch of versions. It did not end up having any effect but I forgot to remove it.